Lesson_8(Dictionary)/HomeWork_8_1.py

```diff
@@ -0,0 +1,18 @@
+#There is a system where there are lists of users.
+#roles = { 'admin' : [...], 'maintainer' : [...], 'manager' : [...], 'developer': [...]} - names should be written in these lists users.
+#When entering a name, it is necessary to check to which role this person belongs and display a message of the type: "Hello, <role>"
+#If the name is not in the lists, output: "Hello, Guest"
+roles = {'admin': [...], 'maintainer': [...], 'manager': [...], 'developer': [...]}
+roles['Quentin'] = 'admin',
+roles['Guillermo'] = 'maintainer',
+roles['Martin'] = 'manager',
+roles['Alfred'] = 'developer'
```

Comment:
This invalid way. You should have already created dictionary
roles = {'admin': ['Quentin'], 'maintainer': ['Guillermo'], 'manager': ['Martin'], 'developer': ['Alfred']}